java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Implement module picker controls for CLM AppStream filters
```

Comment:
is CLM an acronym the customers know of? I would rather have it expanded even if that make a long line.

Reply:
Yes, "CLM" acronym is also mentioned in the docs, so I'd say it's safe to use it here as well